SoptAssignment1/app/src/main/java/com/example/soptassignment1/SignUpActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.soptassignment1
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import com.example.soptassignment1.databinding.ActivitySignUpBinding
+
+
+class SignUpActivity : AppCompatActivity() {
+    private lateinit var binding : ActivitySignUpBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        // 로그인 뷰로 이동
+        binding.btnSignup.setOnClickListener {
+            if(binding.etName.text.isNullOrBlank() || binding.etID.text.isNullOrBlank() || binding.etPassword.text.isNullOrBlank())
+                Toast.makeText(this, "입력되지 않은 정보가 있습니다", Toast.LENGTH_SHORT).show()
+            else {
+                val intent = Intent(this, SignInActivity::class.java)
+                intent.putExtra("ID", binding.etID.text.toString())
+                intent.putExtra("password", binding.etPassword.text.toString())
```

Comment:
선생님 우선 미세먼지 팁을 하나 드리자면.. 좀 더 깔끔하게 표현하기 위해서 두 가지 방법을 추천해드릴건데요.. 우선 첫번째 쉬운 방법 부터 한 번 얘기를 드리자면.. 첫번째로 이렇게 표현이 가능합니다!

```suggestion
                val intent = Intent(this, SignInActivity::class.java)
                                    .putExtra("ID", binding.etID.text.toString())
                                    .putExtra("password", binding.etPassword.text.toString())
```

Reply:
이런 사기 기법!!!!!!